doc/source/ray-core/handling-dependencies.rst

```diff
@@ -335,8 +335,13 @@ The ``runtime_env`` is a Python dictionary including one or more of the followin
 
   - Example: ``{"eager_install": False}``
 
-**Garbage Collection**.  Runtime environment resources on each node (such as conda environments, pip packages, or downloaded ``working_dir`` or ``py_modules`` folders) will be removed when they are no longer
-referenced by any actor, task or job.  To disable this (for example, for debugging purposes) set the environment variable ``RAY_runtime_env_skip_local_gc`` to ``1`` on each node in your cluster before starting Ray (e.g. with ``ray start``).
+Caching and Garbage Collection
+""""""""""""""""""""""""""""""
+Runtime environment resources on each node (such as conda environments, pip packages, or downloaded ``working_dir`` or ``py_modules`` directories) will be cached on the cluster to enable quick reuse across different tasks and actors within a job.  Each field of ``runtime_env``` has its own cache size which defaults to 10 GB.  To change this default, you may set the environment variable ``RAY_RUNTIME_ENV_<field>_CACHE_SIZE_GB`` on each node in your cluster before starting Ray e.g. ``export RAY_RUNTIME_ENV_WORKING_DIR_CACHE_SIZE_GB=1.5``.
```

Comment:
> Each field of ``runtime_env``` has its own cache size which defaults to 10 GB.

I am a bit confused what this means. Does it mean if we have 10 runtime env, we will have 100GB caches?

Reply:
This is valuable feedback, thanks!  I'll edit to make it more clear.  Basically, even if you have 10 runtime_envs, 10GB will be dedicated to the working_dir cache, 10Gb will be dedicated to the `pip` cache, and so on.

Reply:
Updated the doc; please let me know if it's still unclear

Reply:
Hmm why do we have it per runtime env type? This seems a bit odd to me. I think one of the requirements of this feature is to control disk usage, and it seems to be unintuitive.

Reply:
I see. @rkooo567 should we change the design to just have a single unified cache for all fields?  I don't have a concrete use case in mind for why users would need different cache sizes for, say, `working_dir` and `pip`.  It's more flexible, but I think you're right that it comes at the expense of usability.

Reply:
Yeah to me, it seems weird we have per-runtime-env cache (It is also conceptually more confusing). But if this was the design choice that was made during design review, I can live with that. cc @edoakes should we discuss this?